src/main/java/seedu/address/logic/parser/AddElderlyCommandParser.java

```diff
@@ -72,4 +73,15 @@ public AddElderlyCommand parse(String args) throws ParseException {
         Elderly person = new Elderly(name, phone, email, address, nric, age, region, risk, tagList, availableDates);
         return new AddElderlyCommand(person);
     }
+
+    /**
+     * Validates the given ArgumentMultimap by checking that it fulfils certain criteria.
+     *
+     * @param map the ArgumentMultimap to be validated.
+     * @return true if the ArgumentMultimap is valid, false otherwise.
+     */
+    public static boolean validate(ArgumentMultimap map) {
```

Comment:
If you're adding the validate function to all the parsers, could you also add it to the `Parser` interface?

Reply:
Initially, I wanted to enforce this using interface as well but, I realize (soon after) that interfaces cannot have abstract static methods nor can you override static methods in a parent class😢Anyone else has any idea how to enforce this "contract"?

Reply:
Oh yeah it's not possible. Nevermind then.